### ISSUE TITLE
add support for "aria-hidden" attributes

### ISSUE THREAD

**Issue by davelandry** [d3plus/d3plus-text#89]
For hiding whether or not certain texts should be read by screen-readers, it would be great to have a `ariaHidden` method.

**Comment by davelandry** [comment]
This will need to be added to the `<text>` elements themselves: https://github.com/d3plus/d3plus-text/blob/master/src/TextBox.js#L273-L302
